### ISSUE TITLE
Bench Update Fails

### ISSUE THREAD

**Issue by tundebabzy** [frappe/erpnext#9331]
```(python)
Executing erpnext.patches.v8_0.create_domain_docs       #16-05-2017 in crunchy (f3696ac26b13cb7c)
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 91, in <module>
    main()
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 17, in main
    click.Group(commands=commands)(prog_name='bench')
  File "/home/tunde/PycharmProjects/bench/frappe-bench/env/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/env/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/env/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/tunde/PycharmProjects/bench/frappe-bench/env/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/tunde/PycharmProjects/bench/frappe-bench/env/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/env/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/env/local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/commands/__init__.py", line 24, in _func
    ret = f(frappe._dict(ctx.obj), *args, **kwargs)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/commands/site.py", line 216, in migrate
    migrate(context.verbose, rebuild_website=rebuild_website)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/migrate.py", line 31, in migrate
    frappe.modules.patch_handler.run_all()
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/modules/patch_handler.py", line 29, in run_all
    if not run_single(patchmodule = patch):
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/modules/patch_handler.py", line 63, in run_single
    return execute_patch(patchmodule, method, methodargs)
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/modules/patch_handler.py", line 83, in execute_patch
    frappe.get_attr(patchmodule.split()[0] + ".execute")()
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/erpnext/erpnext/patches/v8_0/create_domain_docs.py", line 10, in execute
    frappe.reload_doctype("Domain")
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 650, in reload_doctype
    reload_doc(scrub(db.get_value("DocType", doctype, "module")), "doctype", scrub(doctype),
  File "/home/tunde/PycharmProjects/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 676, in scrub
    return txt.replace(' ','_').replace('-', '_').lower()
AttributeError: 'NoneType' object has no attribute 'replace'
```

**Comment by vishdha** [comment]
frappe.reload_doctype("Domain") cause error 
try frappe.reload_doc("core", "doctype", "docshare") according doctype make changes in it. 
this may work

**Comment by tundebabzy** [comment]
moved to frappe/frappe#3494

**Comment by tundebabzy** [comment]
fixed by #9337
